## 2.4 Transcriptome sequencing

A searchable and exportable tab-separated
[table](Transcriptome_Technical_Metadata.tsv) of the following metadata
is now available.

## Minimal technical metadata for `Transcriptomic FASTQ` file

â€ƒ ğŸ”¹ *italics = potential considerations*

| **metadata**         | **definition**                                                                                     | **examples**                                                  | **source**                                                                                                                             |
|-----------------|---------------------|------------------|-----------------|
| sample_name          | Identifier of the sample                                                                           | e.g.Â ISDsoil1                                                 | **GSC MIxS/MIGS Bacteria** (â€œGSC MIXS: MIGSBacteriaâ€)                                                                                  |
| seq_meth             | Sequencing method used                                                                             | 454 Genome Sequencer FLX \[OBI:0000702\]                      | **GSC MIxS/MIGS Bacteria** (â€œGSC MIXS: MIGSBacteriaâ€), **ENA Metadata Validation: Instrument** (â€œENA Metadata Validation: Instrumentâ€) |
| lib_layout           | Specify whether to expect single, paired, or other configuration of reads                          | e.g.Â single-end, paired end or others                         | **GSC MIxS/MIGS Bacteria** (â€œGSC MIXS: MIGSBacteriaâ€)                                                                                  |
| lib_source           | The lib_source specifies the type of source material that is being sequenced                       | e.g.Â genomic, metagenomic, transcriptomic, etc.               | **ENA Metadata Validation: Source** (â€œENA Metadata Validation: Sourceâ€)                                                                |
| lib_strategy         | Sequencing technique intended for this library                                                     | e.g.Â WGS, Amplicon, etc.                                      | **ENA Metadata Validation: Strategy** (â€œENA Metadata Validation: Strategyâ€)                                                            |
| lib_selection        | Whether any method was used to select and/or enrich the material being sequenced                   | e.g.Â Random, PCR, etc.                                        | **ENA Metadata Validation: Selection** (â€œENA Metadata Validation: Selectionâ€)                                                          |
| nucl_acid_ext        | Literature reference or SOP describing nucleic extraction                                          | e.g.Â CTAB extraction, Phenol-Cloroform Extraction             | **The FAIR Cookbook** (Rocca-Serra et al. 2022)                                                                                        |
| nucl_acid_ext_treat  | Treatment of nucleic acid after extraction                                                         | e.g.Â DNase, RNase                                             | **Adapted from The FAIR Cookbook** (Rocca-Serra et al. 2022)                                                                           |
| ğŸ”¹ *insert_length*   | *for inner dist. calc.* (or incl.Â in lib_layout?)(Array Express parameter: insert â€œNominal Lengthâ€ |                                                               |                                                                                                                                        |
| sequence_count       | Number of reads in the library (sequencing depth) or â€˜spotsâ€™                                       | e.g.Â 32,283,453                                               | **Adapted from NCBI-SRA** (Leinonen et al. 2011)                                                                                       |
| basepairs_count      | Number of base pairs (nucleotides) in the library or â€˜basesâ€™                                       | e.g.Â 6,400,000,000                                            | **Adapted from NCBI-SRA** (Leinonen et al. 2011)                                                                                       |
| average_length       | As basepairs_count divided by sequence_count                                                       | e.g.Â 198                                                      | **Calculated as basepairs_count/sequence_count**                                                                                       |
| sequence_count_qual  | Number of reads in the library (sequencing depth) after quality filtering                          |                                                               | **SRA-Tinder** (NCBI Hackathons)                                                                                                       |
| basepairs_count_qual | Number of base pairs (nucleotides) in the library after quality filtering                          |                                                               | **SRA-Tinder** (NCBI Hackathons)                                                                                                       |
| rRNA treatment       |                                                                                                    | Depletion (Pre-seq, molecular), Removal (Post-seq data), None | **Advances and Challenges in Metatranscriptomic Analysis** (Shakya et al. 2019)                                                        |
| cDNA_ampl_meth       | Technique used to amplify a cDNA library                                                           | e.g.Â MMLV, AMV                                                | **The FAIR Cookbook** (Rocca-Serra et al. 2022)                                                                                        |
| cDNA_oligo           | Type of primer used for cDNA synthesis from RNA                                                    | e.g.Â polyA or random                                          | **The FAIR Cookbook** (Rocca-Serra et al. 2022)                                                                                        |
| checksum             | Hash value for data integrity                                                                      | e.g.Â MD5: cbc41d0e49636872a765b950cb7f410a                    | [Data transfer and data integrity](Data_Transfer_Data_Integrity.md)                                                                    |

| Comments/questions:                                                                                                                                |
|------------------------------------------------------------------------|
| Added insert length, reverse transcriptase, and the primer set as possible technical parameters. Could go under â€œProtocolâ€ parameter. -NME 27APR22 |
| Added cDNA_ampl_meth and cDNA_oligo to the table -MB 11AUG23                                                                                       |

## Minimal technical metadata for `Transcriptomic FASTA` file

| **metadata** | **definition**                                                                                   | **examples**                                                                | **source**                                                                                           |
|-----------------|---------------------|------------------|-----------------|
| run_ref      | Accessions/identifiers linking to the raw data (FASTQ)                                           | e.g.Â accession = â€œERR178314â€                                                | **ENA Submitting Transcriptome Assemblies** (â€œENA Submitting Transcriptome Assembliesâ€)              |
| Locus        | Physical location of a gene on a chromosome                                                      | e.g.Â â€œGAAA01000000 93507 rc mRNAâ€, â€œ11q1.4-q2.1â€                            | **Adapted from ENA Submitting Transcriptome Assemblies** (â€œENA Submitting Transcriptome Assembliesâ€) |
| Description  | Sample description                                                                               | e.g.Â TSA: Latimeria chalumnae voucher 08118, transcriptome shotgun assembly | **Adapted from ENA Submitting Transcriptome Assemblies** (â€œENA Submitting Transcriptome Assembliesâ€) |
| DB Link      | BioProject/Biosample/SRA                                                                         | e.g.Â PRJNA1035062                                                           | **Adapted from ENA Submitting Transcriptome Assemblies** (â€œENA Submitting Transcriptome Assembliesâ€) |
| Reference    | Author/Consortium/Title/Journal                                                                  |                                                                             | **Adapted from ENA Submitting Transcriptome Assemblies** (â€œENA Submitting Transcriptome Assembliesâ€) |
| Seq length   | Number of nucleotides in a transcript. Sequences should be greater than 200 bp in length.        |                                                                             | **Adapted from ENA Submitting Transcriptome Assemblies** (â€œENA Submitting Transcriptome Assembliesâ€) |
| Seq quality  | The accuracy and reliability of the nucleotide sequence information obtained from RNA sequencing |                                                                             | **Adapted from ENA Submitting Transcriptome Assemblies** (â€œENA Submitting Transcriptome Assembliesâ€) |
| Protocols    | Links to protocol accessions                                                                     |                                                                             |                                                                                                      |

| Comments/questions:                                                                                                                        |
|------------------------------------------------------------------------|
| Donâ€™t know which protocol accessions were meant here, was it SOP, and protocol for what, so I left it empty for the time being -MB 11AUG23 |

The publications describing the reasons for formation of **The minimum
information about a genome sequence (MIGS)** (Field et al. 2008),
**Minimum information about a marker gene sequence (MIMARKS) and minimum
information about any (x) sequence (MIxS) specifications** (Yilmaz et
al. 2011) and **Advances and Challenges in Metatranscriptomic Analysis**
(Shakya et al. 2019) can be found online. We also highly encourage the
readers of this GitHub to read **The FAIR cookbook** recipes
(Rocca-Serra et al. 2022), and look for examples of submitted
Transcriptomic samples in **Transcriptome Shotgun Assembly Sequence
Database** (â€œTranscriptome Shotgun Assembly Sequence Databaseâ€).

# References

â€œENA Metadata Validation: Instrument.â€
<https://ena-docs.readthedocs.io/en/latest/submit/reads/webin-cli.html#instrument>.

â€œENA Metadata Validation: Selection.â€
<https://ena-docs.readthedocs.io/en/latest/submit/reads/webin-cli.html#selection>.

â€œENA Metadata Validation: Source.â€
<https://ena-docs.readthedocs.io/en/latest/submit/reads/webin-cli.html#source>.

â€œENA Metadata Validation: Strategy.â€
<https://ena-docs.readthedocs.io/en/latest/submit/reads/webin-cli.html#strategy>.

â€œENA Submitting Transcriptome Assemblies.â€
<https://ena-docs.readthedocs.io/en/latest/submit/assembly/transcriptome.html>.

Field, D., G. Garrity, T. Gray, N. Morrison, J. Selengut, P. Sterk, T.
Tatusova, et al. 2008. â€œThe Minimum Information about a Genome Sequence
(MIGS) Specification.â€ Nature Biotechnology. 2008.
<https://doi.org/10.1038/nbt1360>.

â€œGSC MIXS: MIGSBacteria.â€
<https://genomicsstandardsconsortium.github.io/mixs/MIGSBacteria/>.

Leinonen, R., H. Sugawara, M. Shumway, and International Nucleotide
Sequence Database Collaboration. 2011. â€œThe Sequence Read Archive.â€
*Nucleic Acids Research* 39 (Database issue): D19â€“21.
<https://doi.org/10.1093/nar/gkq1019>.

NCBI Hackathons. â€œ<span class="nocase">SRA-Tinder: A Tool to Discover
Related Sequence Read Archive (SRA) Experiments</span>.â€
<https://github.com/NCBI-Hackathons/SRA_Tinder>.

Rocca-Serra, Philippe, Alasdair J G Gray, Alejandra Delfin Rossaro,
Andrea Splendiani, Andrea Zaliani, Andreas Pippow, Anne Cambon-Thomsen,
et al. 2022. â€œThe FAIR Cookbook.â€
<https://github.com/FAIRplus/the-fair-cookbook/>.

Shakya, Migun, Christopher Quince, James H Campbell, and Zamin K Yang.
2019. â€œAdvances and Challenges in Metatranscriptomic Analysis.â€
*Frontiers in Genetics* 10 (September): 904.
<https://doi.org/10.3389/fgene.2019.00904>.

â€œTranscriptome Shotgun Assembly Sequence Database.â€
<https://www.ncbi.nlm.nih.gov/genbank/tsa/>.

Yilmaz, Pelin et al. 2011. â€œMinimum Information about a Marker Gene
Sequence (MIMARKS) and Minimum Information about Any (x) Sequence (MIxS)
Specifications.â€ *Nature Biotechnology* 29 (5): 415â€“20.
<https://doi.org/10.1038/nbt.1823>.
